OrbitService/CaptureServiceImpl.cpp

```diff
@@ -119,6 +119,34 @@ class GrpcCaptureEventSender final : public CaptureEventSender {
 
 }  // namespace
 
+static void StopTracingHandlerAndCaptureStartStopListenersInParallel(
+    LinuxTracingHandler* tracing_handler,
+    absl::flat_hash_set<CaptureStartStopListener*>* capture_start_stop_listeners) {
+  // LinuxTracingHandler::Stop is blocking, until all perf_event_open events have been processed
+  // and all perf_event_open file descriptors have been closed.
+  // CaptureStartStopListener::OnCaptureStopRequested is also to be assumed blocking,
+  // for example until all CaptureEvents from external producers have been received.
+  // Hence why these methods need to be called in parallel on different threads.
+
```

Comment:
Either remove the empty line here, or move the comment in front of the function.

Reply:
Ok, I can see you don't like vertical space.

Reply:
Hmm, wouldn't say so. But it feels strange to me to have comments not attached to something concretely (either statements, methods, classes,...).